_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+coordinate information, how the data should be structured (e.g. 1 variable per
+file), additional metadata requirements, but also file naming conventions a.k.a.
+the data reference syntax (DRS). All this information is stored in so-called
+CMOR tables. As example, the CMOR tables for the CMIP6 project can be found
+[here](https://github.com/PCMDI/cmip6-cmor-tables).
+{: .callout}
+
+The Earth System Grid Federation (ESGF) is home to all CMIP data. The data
+hosted there typically (mostly) follow the CMIP standards, and therefore
+ESMValTool should work with these data without problems.
+
+Datasets that are *not* part of one of the CMIP projects often don't follow the
+CMOR standards. In this case, a reformatting script can be used to create a
+CMOR-compliant copy of these datasets. CMORizer scripts for several popular
+datasets are included in ESMValTool, and ESMValTool also provides a convenient
+way to execute them.
+
+Occasionally it happens that there are still minor issue with CMIP datasets. In
+those cases, it is possible to fix those issues in ESMValCore before any further
+processing is done. The same can be done for non-CMIP data. The advantage is
+that you don't need to store an additional, reformatted copy of the data. The
+disadvantage is that these fixes should be implemented inside ESMValCore.
+Writing a CMORizer script is technically is simpler.
+
+The concepts discussed so far are illustrated in the figure below.
+![Data flow with ESMValTool](../fig/data_flow.png)
+*Illustration of the data flow in ESMValTool.*
+
+In this lesson, we will re-implement a CMORizer script for the FLUXCOM dataset
+that contains observations of the Gross Primary Production (GPP), a variable
+that is important for calculating components of the global carbon cycle. We will
+go through all the steps and explain relevant topics as we go. If you prefer to
+implement CMOR fixes, please read the documentation
+[here](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/develop/fixing_data.html#fixing-data).
+While fixes are implemented slightly differently, conceptually the process is
+the same and the concepts explained in this episode are still useful.
+
+## Obtaining the data
+
+> ## Get the data
+> The data for this episode is available via the [FluxCom Data
+> Portal](http://www.bgc-jena.mpg.de/geodb/BGI/Home). First you'll need to
+> register. After registration, in the dropdown boxes, select FLUXCOM as the data
+> choice and click download. Three files will be displayed. Click the download
+> button on the "FLUXCOM (RS+METEO) Global Land Carbon Fluxes using CRUNCEP
+> climate data". You'll be send an FTP address to access the server. Connect
+> to the server, follow the path in your email, and look for the file
+> `raw/monthly/GPP.ANN.CRUNCEPv6.monthly.2000.nc`. Download that file and save in
+> in a folder called `/RAWOBS/Tier3/`.
+>
+> Note: you'll need a user-friendly ftp client. On Linux, `ncftp` works okay.
+{: .challenge}
```

Comment:
Why the instruction about getting the data should be in a {:challenge} box while no solution is provided for this challenge? Also, we first teach something and then design a challenge. So users can practice what they have already learned. I suggest making the instruction plain text and remove one of these titles `Obtaining the data` and `Get the data`.

Reply:
Sorry I thought the challenge was also meant to distinguish exercise from theory. I've changed it to plain text.